common-rendering/src/components/liveBlockContainer.tsx

```diff
@@ -129,19 +131,20 @@ const LiveBlockContainer = ({
 				padding: ${space[2]}px ${SIDE_MARGIN_MOBILE}px;
 				margin-bottom: ${space[3]}px;
 				background: ${neutral[100]};
-				border-top: 1px solid ${borderColour};
+				border-top: ${isPinnedPost ? 0 : `1px solid ${borderColour}`};
+				border-bottom: ${isPinnedPost ? 0 : `1px solid ${neutral[86]}`};
```

Comment:
Does it also work if we make the whole line conditional? So we only set a border if not a pinned post. Of course, this idea falls over if the `0` is needed here